README.md

```
@@ -48,52 +48,93 @@ pnpm prisma generate
+
+Required environment variables:
+
+```env
+AUTH_SECRET=        # generate with: npx auth secret
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+```
+
+Configure your Google OAuth app at [console.cloud.google.com](https://console.cloud.google.com) and set the redirect URI to:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)